docs/_posts/2018-01-23-grep-on-the-fly-in-spacevim.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Grep on the fly in SpaceVim"
+categories: feature
+excerpt: "Run grep Asynchronously and on the fly as you type in SpaceVim, quick searching whole project or opened files."
+image: https://user-images.githubusercontent.com/13142418/35278709-7856ed62-0010-11e8-8b1e-e6cc6374b0dc.gif
+---
+
+# Asynchronous grep on the fly
+
+FlyGrep means **grep on the fly**, it will update the result as you type. Of course, it is running
+asynchronously. Before using this feature, you need to install a searching tool. FlyGrep can be
+interfaced with different searching tools like: `ag`, `rg`, `ack`, `pt` and `grep`, select any one
+you like.
+
+This ia a build-in plugin in SpaceVim, and we also separate a plugin : [FlyGrep.vim](https://github.com/wsdjeg/FlyGrep.vim)
+
+## Features
+
+- **Searching in a project**
+
+In SpaceVim, you can use `SPC s p` or `SPC s /` to searching in current project.
+
+![searching project](https://user-images.githubusercontent.com/13142418/35278709-7856ed62-0010-11e8-8b1e-e6cc6374b0dc.gif)
+
+- **Searching in current file**
+
+You can use `SPC s s` to searching in current file. To searching word under the cursor, you may need to press `SPC s S`.
```

Comment:
- `to searching` ==> `to search`
- `search the current file` sounds more appropriate for me.
- `you may need to press` ==> `you can press`. (I like it to be simple.)
- `searching word` ==> `search words`. It applies to the [Line 33](https://github.com/SpaceVim/SpaceVim/pull/1314/files#diff-9d8ec7295244b680109134cbc05357ceR33)